analyses/molecular-subtyping-pathology/02-incorporate-clinical-feedback.Rmd

```diff
@@ -0,0 +1,200 @@
+PT_0MXPTTM3 | H3 WT | Initial CNS Tumor
+PT_M9XXJ4GR | H3F3A K28M | Initial CNS Tumor
+PT_KTRJ8TFY | H3F3A K28M | Initial CNS Tumor
+PT_KTRJ8TFY | no report | Progressive   Disease Post-Mortem
+PT_1E3E6GMF | H3F3A K28M | Initial CNS Tumor
+PT_VPEMAQBN | HIST1H3B K28M | Initial CNS Tumor
+PT_7P6J57H3 | H3F3A K28M | Initial CNS Tumor
+PT_C5FKRB1P | H3 WT | Initial CNS Tumor
+PT_CSKHQB16 | HIST1H3B K28M | Initial CNS Tumor
+PT_1AAYYGGY | H3F3A K28M | Initial CNS Tumor
+PT_NWYSD53S | H3F3A K28M | Initial CNS Tumor
+PT_8P368R5B | No report | Initial CNS Tumor
+PT_C9YDTZPA | H3F3A K28M | Initial CNS Tumor
+PT_KAQMYFYB | H3F3A K28M | Initial CNS Tumor
+PT_Y74CVASJ | H3F3A K28M | Initial CNS Tumor
+PT_W5GP3F6B | H3F3A K28M | Initial CNS Tumor
+PT_A06JR0E5 | H3F3A K28M | Initial CNS Tumor
+PT_EN2RN5Y1 | H3F3A K28M | Initial CNS Tumor
+PT_DMAF1J4A | H3F3A K28M | Initial CNS Tumor
+PT_RYMG3M91 | H3F3A K28M | Initial CNS Tumor
+PT_RE6AXQM1 | H3F3A K28M | Initial CNS Tumor
+PT_G16VK7FR | HIST1H3B K28M | Initial CNS Tumor
+PT_1YQH5NSH | H3F3A K28M | Initial CNS Tumor
+
+## Set up 
+
+```{r}
+# This logic is the same as analyses/gene-set-enrichment-analysis/02-model-gsea.Rmd
+# Assigning params$is_ci to running_in_ci avoids a locked binding error
+running_in_ci <- params$is_ci
+
+# Are we testing? In case of a non 0/1 number, we recast as logical, and then 
+# ensure logical.
+if (running_in_ci %in% c(0,1)) running_in_ci <- as.logical(running_in_ci)
+if (!(is.logical(running_in_ci)))
+{
+  stop("\n\nERROR: The parameter `is_ci` should be FALSE/TRUE (or 0/1).")
+}
+```
+
+
+### Directory and files
+
+#### Directories
+
+```{r}
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+results_dir <- "results"
+```
+
+
+#### Input
+
+When we run this locally, we want to tie it to a specific version of the histologies file _prior_ to reviewed clinical subtypes for PNOC003 samples and pathology feedback [#751](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/751)
+
+```{r}
+if (running_in_ci) {
+  # clinical pbta histology
+clinical <- readr::read_tsv(file.path(data_dir, 
+                                "pbta-histologies.tsv")) 
+} else {
+    # clinical pbta histology
+clinical <- readr::read_tsv(file.path(data_dir, "release-v15-20200228",
+                                "pbta-histologies.tsv")) 
+}
+```
+
+
+```{r}
+library("tidyverse")
+
+# clinical review file for PNOC003 
+pnoc003_review <- readr::read_tsv(file.path("input","pnoc003-clinical-review.tsv"))
+  
+# subtyping from molecular-subtype-HGG
+subtype <- readr::read_tsv(file.path(results_dir,
+                         "compiled_molecular_subtypes.tsv")) %>%  
+  # ids to match
+  left_join(clinical[,c("Kids_First_Participant_ID",
+                        "Kids_First_Biospecimen_ID",
+                        # need tumor_descriptor to match to clinical review
+                        "tumor_descriptor")],by=c("Kids_First_Biospecimen_ID",
```

Comment:
I think this is the only spot where we use the `pbta-histologies.tsv` file, is that correct? If so, I don't think we need to tie this to a specific release.

Reply:
thanks for the review, will remove the specific release chunk

Reply:
hmm there seem to be some updates to tumor descriptor values from v15 to v17, that should be ok?

Reply:
Ah, yes, that should be okay. I think we can think of the v17 tumor descriptor values as more correct!